curvefs/src/mds/fs.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * @Project: curve
+ * @Date: 2021-06-04 14:11:16
+ * @Author: chenwei
+ */
+
+#include "curvefs/src/mds/fs.h"
+namespace curvefs {
+namespace mds {
+MdsFsInfo::MdsFsInfo(uint32_t fsId, std::string fsName, FsStatus status,
+                    uint64_t rootInodeId, uint64_t capacity,
+                    uint64_t blockSize, const Volume& volume) {
+    fsId_ = fsId;
```

Comment:
move those assignment to initializer list

Reply:
done